install.php

```diff
@@ -1,3 +1,40 @@
 <?php
-$_GET["mod"] = "install";
-include_once("index.php");
+
+//raise error reporting - could be interesting and this is autmomatically reset when index.php is executed
+error_reporting(E_ALL);
+
+
+/**
+* Check for common failures
+*
+* Small function to check against a number of dependencies that cause install.php to fail without showing any error details
+* @return bool check result. True if error found, false if all OK
```

Comment:
Afaik PHPDoc says, that we insert a blank line between the descriptions and the parameters/return values.